pkg/backup/targets.go

```diff
@@ -159,7 +161,9 @@ func getRelevantDescChanges(
 		} else if change.Desc != nil {
 			desc := backupinfo.NewDescriptorForManifest(change.Desc)
 			switch desc := desc.(type) {
-			case catalog.TableDescriptor, catalog.TypeDescriptor, catalog.SchemaDescriptor:
+			case catalog.TableDescriptor, catalog.TypeDescriptor, catalog.SchemaDescriptor, catalog.FunctionDescriptor:
```

Comment:
Are we also missing Database here?

Reply:
ugh, we are. i will address in a separate patch as the fix there is not as trivial for _reasons_

Reply:
er nvm. one bug is hard to solve and one is easy. i will solve the easy bug with a patch to this PR.